services/core-api/app/api/mines/alerts/models/mine_alert.py

```diff
@@ -17,6 +19,10 @@ class MineAlert(SoftDeleteMixin, AuditMixin, Base):
 
     end_date = db.Column(db.DateTime)
 
+    mine_table = db.relationship('MineMapViewLocation', lazy='joined', primaryjoin='and_(foreign(MineAlert.mine_guid) == remote(MineMapViewLocation.mine_guid))')
+    mine_name = association_proxy('mine_table', 'mine_name')
+    mine_no = association_proxy('mine_table', 'mine_no')
+
```

Comment:
I waffled over this _a lot_.
The reason I'm using MineMapViewLocation vs Mine is that joining _Mine_ results in several other tables being join, but MineMapViewLocation is a view of just a few columns from the mine table (so only one join). Did not add the mine name/no to the view, it was already there, but I did add those fields to the python model.